src/models/ClassPotentialC2HDM.cpp

```diff
@@ -2396,133 +2328,590 @@ std::vector<double> Class_Potential_C2HDM::calc_CT() const
   return parCT;
 }
 
-void Class_Potential_C2HDM::TripleHiggsCouplings()
+/**
+ * Ensures the correct rotation matrix convention
```

Comment:
The documentation should always be in the header file.
In this case the origin class should have it and just pass it through to the overrides, unless they need to specify it.